modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/java/com/liferay/frontend/data/set/views/web/internal/display/context/FDSViewsDisplayContext.java

```diff
@@ -58,6 +58,21 @@ public FDSViewsDisplayContext(
 		_fdsEntryObjectDefinition =
 			objectDefinitionLocalService.fetchObjectDefinition(
 				_themeDisplay.getCompanyId(), "FDSEntry");
+		_fdsViewObjectDefinition =
+			objectDefinitionLocalService.fetchObjectDefinition(
+				_themeDisplay.getCompanyId(), "FDSView");
+	}
+
+	public String getEditDataSetURL() {
+		return PortletURLBuilder.create(
+			PortletURLFactoryUtil.create(
+				_renderRequest, FDSViewsPortletKeys.FDS_VIEWS,
+				RenderRequest.RENDER_PHASE)
+		).setMVCPath(
+			"/data_set.jsp"
+		).setBackURL(
+			_themeDisplay.getURLCurrent()
+		).buildString();
```

Comment:
The main new naming convention is `DataSet`. It is like that in all code, all the way up to persistence.

All naming related to the module is the same, like here `FDSViewsPortletKeys`. That should not be changed unless we move everything to a standalone app.